server/PokerTableObjects/Card.js

```diff
@@ -0,0 +1,38 @@
+// this class will most likely be removed from the final design, it was built primarly for testing the
+// intial poker hand rank calculator which had its own UI
+module.exports = class Card {
+    constructor(number, suit) {
+        this.number = number;
+        this.suit = suit;
+    }
+    
+    static getSuit(number){
+        if(number === 1){
+            return 'S';
+        }
+        else if(number === 2){
+            return 'C';
+        }
+        else if(number === 3){
+            return 'H';
+        }
+        else{
+            return 'D';
+        }
+    }
```

Comment:
Is it worth turning these into enums?

Reply:
Or a switch statement at least. I've commented on this in previous PR. Has not been addressed.

Reply:
updated to switch statements

Reply:
function actually was not being used, removed entirely